UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Index.cshtml

```diff
@@ -15,7 +15,16 @@
 
 <div class="tab-content" id="nmTabContent">
     <div class="tab-pane fade show active" id="weekly" role="tabpanel" aria-labelledby="weekly-tab">
-
+        @if (@ViewBag.IsDistributor)
+        {
+            <div id="NMInfo" class="alert alert-primary p-4" role="alert">
+                <i class="bi bi-exclamation-circle-fill" aria-hidden="true"></i>
+    <div class="alert-head">Temporary Notice</div>
+    <p>
+        Redirecting to download the Weekly tracing file due to unavailability in MSI at the moment :<br>
+         <td><a href="~/nms/tracings/Title_and_Index_2022.pdf" target="_blank">Weekly Tracings File</a></td>
```

Comment:
Please remove <td> tag as its not required here. Element 'td' cannot be nested inside element 'p'.
Also, please correct the alignment for the whole @if block.